### DISCUSSION TITLE
CI fail with generics: expected ';', found '{')]]"

### DISCUSSION THREAD

**Discussion by michelececcacci** [golangci/golangci-lint#3556] [Q&A] [answered]
Same result both go 1.18 and go 1.19. This line apparently makes the whole CI fail. The project builds, and unit test pass, so i don't think there are errors on my side. 
```go
	var rw = ResultWrapper[BlogEntry]{}
```
[Link to a run that failed](https://github.com/michelececcacci/codeforces/actions/runs/4082734721/jobs/7037430530)

**Comment by ldez** [comment, marked as answer]
hello,

I don't reproduce the problem locally.

**Reply by michelececcacci** [reply]
Hi, i also tried to reproduce locally (go 1.18.1) and it's not failing. I will try to find a solution and update, maybe this can be also useful to someone else.

**Comment by michelececcacci** [comment]
Turns out it was a problem on my side, and it went away by rewriting my .yml file.
